### PR TITLE
Dabonnie/fork test

### PR DESCRIPTION
**Pull request by dabonnie**
This PR provides integration testing for the following:

### scheduled
 * the latest released colcon-bundle
 * the latest git master colcon-bundle

### pull request
 * colcon-bundle installed via the checked out workspace, which triggered the PR
 * the above scheduled jobs

### CODE REVIEW

**Review by zmichaels11 — CHANGES_REQUESTED** [review]

File: .github/workflows/test_bundle.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+name: colcon-bundle Integration Test
+on:
+  pull_request:
+  schedule:
+    - cron: '0 * * * *'
+
+# Test the released and current master branch
```
Comment (prajakta-gokhale) [inline .github/workflows/test_bundle.yml:7]: Do we need to test released and current master branch for all pull requests?

File: .github/workflows/test_bundle.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        id: build-from-repos
+        with:
+          vcs-repo-file-url: "${{ github.workspace }}/.github/workflows/test_bundle.repos"
+          package-name: ros_tutorials navigation
+          source-ros-binary-installation: kinetic
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install python3-apt python3-pip
+          sudo -H pip3 install -U setuptools==44.0.0
```
Comment (zmichaels11) [inline .github/workflows/test_bundle.yml:37]: Is it mentioned anywhere why `setuptools` is pinned to version `44.0.0`?

**Review by zmichaels11 — APPROVED** [review]
LGTM with green CI

File: .github/workflows/test_bundle.yml (hunk 1 above)
Comment (dabonnie) [reply .github/workflows/test_bundle.yml:7]: Yes: if a contribution changes the vcs repos file (colcon-bundle/.github/workflows/test_bundle.repos) we won't catch breaking changes until merged.

File: .github/workflows/test_bundle.yml (hunk 2 above)
Comment (dabonnie) [reply .github/workflows/test_bundle.yml:37]: I followed the [test.yml](https://github.com/colcon/colcon-bundle/blob/master/.github/workflows/test.yml#L25) usage. Should we install the latest? If so, can fix here and let's file an issue to address the unit tests.

**Review by prajakta-gokhale — APPROVED** [review]

**Pull request merged**